### ISSUE TITLE
error 709 when changing computers with delegated harvesting active

### ISSUE THREAD

**Issue by jabo38** [NewEconomyMovement/NemCommunityClient#394]
when I set up delegated harvesting on computer 1 everything works fine.  I harvest as much as I want.  

But then when I import that private key onto a different computer and try to harvest from computer 2, then it won't let me and I get error 709, that my account is unknown and needs to be a part of one transaction.  I think send to or from that account doesn't help.  Only deactivating remote harvesting and then reactivating it helps.

**Comment by fyahfox** [comment]
I dont fully understand all details, so let me ask some questions:

With "set up delegated harvesting" you mean activating it, and then starting it while connected to the local NIS on computer 1?

And which priv key did you import to a different computer? I guess the actual account's priv key, not the remote account's priv key? And then, did it show that delegated harvesting was activated and you started that (while connected to the local NIS on that different computer now)?

**Comment by Jaguar0625** [comment]
I think this is because there is no way to import a remote harvesting private key at the moment.

**Comment by fyahfox** [comment]
Aren't remote accounts deterministic till now? So there is no need to import a remote account's priv key. Is already there, when you import an account...

**Comment by Jaguar0625** [comment]
Right now, NCC creates a random remote private key for each account being used remotely. Since it's random, there's no way to derive the remote private key from the account private key. The local - remote account relationship is stored in the blockchain, but the keys are not.

II don't think @gimer changed this.

**Comment by fyahfox** [comment]
Just checked it, Jag is right. The remote account's priv key changes when you import the original account's priv key to a new wallet/NCC.

**Comment by jabo38** [comment]
since it wasn't really my fault then and I couldn't discern that from the error message, I guess my suggestion would be to add a part about delegated harvesting accounts not being able to be imported properly.  The error kept on telling me I didn't make a transaction so I kept on send back and forth to myself.  :-)  What actually worked was deactivated delegated harvesting and then reactivating it again.

**Comment by fyahfox** [comment]
Yes, the activation is what makes the account "known", but when you do this, the other computer can't use delegated harvesting for that account again.

So we should think about a good solution for this problem/usecase.
Giving the user the chance to import a remote account's priv key is one way, but it seems complicated.
